sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/job_definition.py

```diff
@@ -0,0 +1,52 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict, Union, Any
+
+from azure.ai.ml._schema._deployment.batch.job_definition_schema import JobDefinitionSchema
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+from azure.ai.ml.entities._component.component import Component
+
+
+class JobDefinition:
+    """Job Definition entity
+
+    :param job: Job definition
+    :type job: Union[Component, str]
```

Comment:
It's a bit weird to me that the class is called `JobDefinition` and we have a param called "job" described as also "Job definition" but it's of a different class `Component`.
How can we simplify these concepts (arg name or description changes? I assume the type is correct?)

Reply:
Yes, that was wrong. The right type was `Job`. Thanks for catching that